prowler/lib/outputs/slack.py

```diff
@@ -0,0 +1,137 @@
+import sys
+
+from slack_sdk import WebClient
+
+from prowler.config.config import aws_logo, azure_logo, gcp_logo, square_logo_img
+from prowler.lib.logger import logger
+
+
+def send_slack_message(token, channel, stats, provider, audit_info):
```

Comment:
Could you please include a test for this function?
- One mocking success
- One mocking some failure/exception to handle that behaviour

Thanks!